Assets/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -3975,7 +3975,7 @@ public void Devices_AreMadeCurrentWhenReceivingStateEvent()
 
         Assert.That(Gamepad.current, Is.Not.SameAs(gamepad1));
 
-        InputSystem.QueueStateEvent(gamepad1, new GamepadState());
+        InputSystem.QueueStateEvent(gamepad1, new GamepadState().WithButton(GamepadButton.A));
```

Comment:
Ideally, also test the opposite case here. That just sending a state event only shouldn't result in a change.